bitrise.yml

```diff
@@ -51,6 +51,17 @@ workflows:
         - notify_user_groups: $NOTIFY_USER_GROUPS
         - notify_email_list: $NOTIFY_EMAIL_LIST
         - is_enable_public_page: "true"
+    - path::./:
+        title: IOS Test xcarhive
+        run_if: true
+        inputs:
+        - build_url: $BITRISE_BUILD_URL
+        - build_api_token: $BITRISE_BUILD_API_TOKEN
+        - is_compress: "false"
+        - deploy_path: ./archives
+        - notify_user_groups: $NOTIFY_USER_GROUPS
+        - notify_email_list: $NOTIFY_EMAIL_LIST
+        - is_enable_public_page: "true"
```

Comment:
please add xyz.xcarchive.zip to the sample repo archives subdir to be able to test this functionality
https://github.com/bitrise-io/sample-artifacts/tree/master/archives

Reply:
Fixed.